config/openshift/cryostat-quickstart.yaml

```diff
@@ -7,86 +7,116 @@ metadata:
-3. Click the external link icon to open the URL and run the Cryostat web application in a new browser tab. 
-4. Enter your Red Hat OpenShift credentials to log in to the Cryostat web application. <br />
-5. In the web application, click **Recordings** in the navigation menu. <br />
-6. View quarkus-quickstarts (or your deployed application) in the **Targets** list. <br /><br />
-Your Java application has now been configured to be detected by Cryostat. To profile your Java applications with Cryostat, please follow [this documentation.](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/getting_started_with_cryostat/assembly_creating-recordings_assembly_configuing-java-applications)"
-    title: "View your Java Application as a target in Cryostat"
+      failedTaskHelp: >-
+        This task isn't verified yet. Try the task again, or [read
+        more](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/getting_started_with_cryostat/assembly_configuing-java-applications_cryostat)
+        about this topic.
+      instructions: >
+        #### Click **Reload** at the bottom of the **Environment** tab.
+
+        Do you see a **single values entry** for **`JAVA_OPTS_APPEND`**?
+  - description: >
+      To update Red Hat OpenShift Service with the remote JMX port:
+
+      1. In the main navigation, click the dropdown menu and select **Administrator**.
+
+      1. In the navigation menu, click **Networking > Services**.
+
+      1. Click on the quarkus-quickstarts service name (or your application's service name).
+
+      1. Click the “YAML” tab to open the YAML editor.
+
+      1. In the **spec > ports** section, add a new port for jfr-jmx
+          ```
+          - name: jfr-jmx
+            protocol: TCP
+            port: 9091
+            targetPort: 9091
+          ```
+
+      1. Click **Save**.
+    title: Updating Red Hat OpenShift Service with the remote JMX port
+    review:
+      failedTaskHelp: >-
+        This task isn't verified yet. Try the task again, or [read
+        more](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/getting_started_with_cryostat/assembly_configuing-java-applications_cryostat)
+        about this topic.
+      instructions: >
+        #### Verify that the service port is added to the sample resource:
+
+        1. In the main navigation, click the dropdown menu and select **Developer**.
+
+        1. In the navigation menu, click **Topology**.
+
+        1. Click on the quarkus-quickstarts deployment name (or your application's service name).
+
+        1. Click on the **Resources** tab.
+  - description: >
+      View your Java application as a target in the Red Hat build of Cryostat web application:
+
+      1. Click on the perspective switcher at the top of the navigation, and select **Developer**.
```

Comment:
Should be " 1. In the main navigation, click the dropdown menu and select **Developer**. "

Reply:
Fixed